packages/father-build/src/fixtures/build/rollup-esm-file/.fatherrc.js

```diff
@@ -0,0 +1,7 @@
+
+export default {
+  esm: { 
+    type: 'rollup', 
+    file: 'demo' 
```

Comment:
看之前的 [case](https://github.com/umijs/father/blob/8a39ea4e5c52e980aee7a1bbed4ef1fa6d126705/packages/father-build/src/fixtures/build/rollup-outputFilename-with-overrides/.fatherrc.js#L17) 这里的 esm 后缀应该是要手动控制的，之前说错了

```suggestion
    file: 'demo.esm' 
```